katdal/vis_flags_weights.py

```diff
@@ -192,17 +216,21 @@ class ChunkStoreVisFlagsWeights(VisFlagsWeights):
         Chunk store
     chunk_info : dict mapping array name to info dict
         Dict specifying prefix, dtype, shape and chunks per array
-    corrprods : sequence of 2-tuples of input labels
-        Correlation products. If given, the weights for baseline (inp1, inp2)
-        will be divided by the square root of the product of the corresponding
-        autocorrelations vis[inp1,inp1] and vis[inp2,inp2].
+    corrprods : sequence of 2-tuples of input labels, optional
+        Correlation products. If given, the raw weights for baseline (inp1, inp2)
+        will be divided by the product of the corresponding autocorrelations
+        vis[inp1,inp1] and vis[inp2,inp2], and the raw weights will also
+        be exposed as `unscaled_weights`.
+    raw_weights_are_unscaled : bool, optional
+        False if raw weights are already scaled by autocorrelations, in which
```

Comment:
Describing the meaning when a negated variable name (unscaled) is false is a double-negative and confusing.

Reply:
Point taken. I've completely reworked this - now called `stored_weights_are_scaled`. Nice and positive.